src/templates/components/account_sections/profile_header_section.html

```diff
@@ -2,7 +2,7 @@
     {% if user.company.cover_image %}
         <img src="{{ user.company.cover_image }}" class="w-full h-full object-contain" />
     {% endif %}
-    <form method="POST" action="/api/v1/company/{{ user.company.id }}/cover" id="upload-cover-form" enctype="multipart/form-data">
+    <form method="POST"
```

Comment:
This will break the app. You removed something thats used

Reply:
accident